.travis.yml

```diff
@@ -9,11 +9,18 @@ env:
 
 install: true
 
+before_script:
+  - cd /tmp && mkdir tools && cd -
+  - sudo curl -s -L "https://github.com/sonatype-nexus-community/nancy/releases/download/v0.0.21/nancy-linux.amd64-v0.0.21" -o "/tmp/tools/nancy"
```

Comment:
why sudo?

Reply:
copied from the official example, but you're right it's not necessary